app/views/Dashboard/Country/index.tsx

```diff
@@ -612,6 +613,7 @@ function Country(props: Props) {
                 id: total.id,
                 emergency: total.emergency,
                 totalCases: total.indicatorValue,
+                format: total.format as FormatType,
```

Comment:
I don't think this cast is necessary. Please check.

Reply:
Its needed